package.json

```diff
@@ -60,7 +60,8 @@
   "bin": {
     "hsd": "./bin/hsd",
     "hsd-node": "./bin/node",
-    "hsd-spvnode": "./bin/spvnode"
+    "hsd-spvnode": "./bin/spvnode",
+    "hswallet": "./bin/hsw"
```

Comment:
nit: in order to follow the naming conventions of other libraries, can you change this binary to `hs-wallet`?

Reply:
done! thanks.